frontend/appflowy_flutter/lib/plugins/document/presentation/editor_page.dart

```diff
@@ -250,7 +250,10 @@ class _AppFlowyEditorPageState extends State<AppFlowyEditorPage> {
     final customBlockComponentBuilderMap = {
       PageBlockKeys.type: PageBlockComponentBuilder(),
       ParagraphBlockKeys.type: ParagraphBlockComponentBuilder(
-        configuration: configuration,
+        configuration: configuration.copyWith(
+          placeholderText: (_) => 'Start Typing',
```

Comment:
The placeholder text should only be visible when the document is on the row detail page, so you can't define a placeholder here that will make all the documents display this hint text.

And please localize the hint text.